### ISSUE TITLE
Understanding bootstrap sample size requirements for bcajack2

### ISSUE THREAD

**Issue by tcovert** [bnaras/bcaboot#1]
Since `bcajack2` internally runs a regression of the bootstrap estimates onto the bootstrap sample matrix, is it correct to say that the *minimum* number of bootstrap samples needed for this routine to work is roughly `(1/pct) * nrow(X)`?  The `(1/pct)` factor is to account for the fact that only a subset of the bootstrap samples get used.

additionally, for documentation purposes, its worth pointing out that even when one has constructed `Blist <- list(Y = Y, tt = tt, t0=t0)`, the documentation's stated function call of `bcajack2(Blist)` does not seem to work, while `bcajack2(B = Blist)` does.

Thanks for developing this package - aside from these questions above I have found it to be awesome!

**Comment by bnaras** [comment]
Yes, your remark on min sample size is correct. However, the `jacksd` values of internal accuracy take this into account. 

`bcajack2` needs more work and is sort of beta. Thanks for your report. We'll fix in the subsequent versions.

**Comment by tcovert** [comment]
Again, if you are interested in bug reports, in the problem I am using `bcajack2` with, N = 2280, I'm doing 7000 bootstraps, and I get NaN for the `jacksd` values of the BCa points.  As far as I can tell, none of the bootstrap values in `tt` are NaN, so my guess is that this may have something to do with N vs. the number of bootstraps I am running.

**Comment by bnaras** [comment]
Can you post a reprex so that I may check?
